brownfield_django/main/views.py

```diff
@@ -235,7 +236,8 @@ def get_queryset(self):
         up = self.request.user.profile
         queryset = User.objects.none()
         if up.is_admin():
-            instructors = UserProfile.objects.filter(profile_type='TE')
+            instructors = UserProfile.objects.filter(
+                profile_type='TE').filter(archive=False)
```

Comment:
Just curious, are you wanting an "OR" selector from the database or an "AND"? I have chained filters before with unexpected results... http://stackoverflow.com/questions/8164675/chaining-multiple-filter-in-django-is-this-a-bug